### DISCUSSION TITLE
Mappin tool

### DISCUSSION THREAD

**Discussion by MLocria** [axelboc/anki-ultimate-geography#615] [Q&A] [answered]
¡Hello, everyone!

I hope it's ok to open a discussion on this but I was curious as to what tool is being used to generate all the maps in this deck. I'm trying to create a more detailed deck for my country (specifically, the water areas) and I'd love to replicate the simple and effective maps you have used here.

Thanks in advanced.

**Comment by aplaice** [comment, marked as answer]
Hi!

Most of the maps that we have came from Wikipedia (Wikimedia), from the ["Kartenwerkstatt"](https://de.wikipedia.org/wiki/Wikipedia:Kartenwerkstatt/Positionskarten) (for reasons I don't understand a very large portion of the Wikipedia/Wikimedia map-creation community is German, but the English link to the community/project is [here](https://en.wikipedia.org/wiki/Wikipedia:Graphics_Lab/Map_workshop)).

The remaining maps were either:

1. Generated manually by combining two-or-more maps (i.e. existing images) from Wikipedia, in order to add a "zoombox" and/or change cropping.

    This was usually done with a simple vector image-editing program (e.g. inkscape), though in some cases we went went slightly "overboard" (e.g. [here](https://github.com/anki-geo/ultimate-geography/issues/461)) in order that the projections of the two maps were the same (which looks nicer/is technically more correct).

2. Generated from [Natural Earth Data](https://www.naturalearthdata.com/) using [Mapshaper](https://github.com/mbloch/mapshaper).

    For some of the maps this was done manually.  For some others (mainly the sea maps) this was semi-automated using [this](https://github.com/aplaice/svg_sea_maps) set of very ugly and messy scripts.  However, in all cases the core was still just shapefiles from Natural Earth and Mapshaper (plus some styling changes to the resulting SVGs, for consistency).

Where possible we tried to follow the style of the above-mentioned Wikipedia location map scheme, and uploaded our maps to Wikimedia in case they were useful to other people.

<hr/>

In other words, unless you already have a set of images (maps) that you can adapt by cropping/zooming/etc., you need:

1. A set of source data (often "shapefiles").

    For relatively large-scale maps Natural Earth is great.  For water bodies it does have [some data](https://www.naturalearthdata.com/downloads/10m-physical-vectors/), but I'm not sure if the detail is sufficient to product nice-looking maps on a local/country scale.

2. A program for converting the source data into nice images.

    I've had a great experience with mapshaper, which has both an interactive and a batch mode, but there are other options.

<hr/>

Hope this is useful and good luck!  Feel free to ask if you have any queries!  (You might also want to ask at the [Wikipedia Graphics Lab](https://en.wikipedia.org/wiki/Wikipedia:Graphics_Lab/Map_workshop), where there might be more qualified people.)

**Reply by MLocria** [reply]
You, internet person, are an absolute legend :clap: . Thank you so much! I'll go do my homework and pop back here if I have questions. Have a wonderful holiday season! :star2:
